src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -43,6 +42,17 @@ internal class HttpTriggerAttributeBindingProvider : ITriggerBindingProvider
         internal const string RequestBindingName = "$request";
 
         private readonly Action<HttpRequest, object> _responseHook;
+        private static readonly Type[] _supportedTypes = new Type[]
```

Comment:
This is not required, but moved this here as it will be built ahead of time. The goal was to remove the dependency on the `StreamValueBinder`, which would have made this required, but I'm not doing that in this PR.

Reply:
I thought your plan was to cross compile these types rather than copy code? However since StreamValueBinder/ValueBinder are public, that will result in conflicts if people pull in multiple extensions?